Makefile

```diff
@@ -20,34 +25,35 @@ init:
 
 lint:
 	@echo "Running flake8..."
-	flake8 moto tests
+	flake8 moto/${SERVICE_NAME} tests/${TEST_SERVICE}
 	@echo "Running black... "
 	$(eval black_version := $(shell grep "^black==" requirements-dev.txt | sed "s/black==//"))
 	@echo "(Make sure you have black-$(black_version) installed, as other versions will produce different results)"
-	black --check moto/ tests/
+	black --check moto/${SERVICE_NAME} tests/${TEST_SERVICE}
 	@echo "Running pylint..."
-	pylint -j 0 moto tests
+	pylint -j 0 moto/${SERVICE_NAME} tests/${TEST_SERVICE}
 	@echo "Running MyPy..."
-	mypy --install-types --non-interactive
+	mypy --install-types --non-interactive moto/${SERVICE_NAME}
```

Comment:
Not all services support typing at the moment - the ones that do are listed in `setup.cfg`. This is a very quick check, it only takes a few seconds, so I would suggest to just leave this as is.

Reply:
changed it a bit, but we can go this route as well.